ragaai_catalyst/redteaming/red_teaming.py

```
@@ -0,0 +1,313 @@
+        """
+
+        # Validate detectors
+        inbuild_detector = []
+        for detector in detectors:
+            if type(detector) == str:
+                inbuild_detector.append(detector)
+            elif type(detector) == dict:
+                if 'custom' not in detector.keys() or len(detector.keys()) != 1:
+                    raise('The custom detector must be a dictionary with only key "custom" and a string as a value')
+            else:
+                raise('Detector must be a string or a dictionary with only key "custom" and a string as a value')
```

Ruff (0.8.2)

Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)